app/lib/services/voip/MediaSessionInstance.ts

```
@@ -14,6 +15,7 @@ import { mediaSessionStore } from './MediaSessionStore';
 import { useCallStore } from './useCallStore';
 import { store } from '../../store/auxStore';
 import sdk from '../sdk';
+import { mediaCallsStateSignals } from '../../services/restApi';
```

ESLint

[error] Useless path segments for "../../services/restApi", should be "../restApi"

(import/no-useless-path-segments)